global.R

```diff
@@ -68,7 +68,7 @@ ordered_tabs <- c(
 
 tabs <- list(
   QuestionTab$new("title", NULL, "auth", FALSE, FALSE),
-  QuestionTab$new("auth", "title", NULL),
+  QuestionTab$new("auth", "title", NULL, ui_pars = list(captcha_code=read_yaml('secret.yml')$captcha_code)),
```

Comment:
can we avoid having to read the yaml file twice?  I was expecting this to need to be done dynamically.

Reply:
We can avoid reading the yaml second time by reading its content once outside the server ( => in the global). Unless there are (security?) arguments not to use such global variables for secret variables

Reply:
thanks a lot, I'm happy with globals approach